crates/bevy_render/src/mesh/shape/cone.rs

```diff
@@ -0,0 +1,137 @@
+use crate::mesh::{Indices, Mesh};
+use wgpu::PrimitiveTopology;
+
+/// A cone which stands on the XZ plane
```

Comment:
```suggestion
/// A cone which stands on the XZ plane.
///
/// If either radius is 0., the tip will be sharp. Otherwise, the cone will appear truncated, with flat surfaces on either end.
```